signup-service/src/main/java/io/mosip/signup/services/WebSocketHandler.java

```diff
@@ -79,6 +87,13 @@ public void processFrames(IdentityVerificationRequest identityVerificationReques
         dto.setStepCode(identityVerificationRequest.getStepCode());
         dto.setFrames(identityVerificationRequest.getFrames());
         plugin.verify(identityVerificationRequest.getSlotId(), dto);
+        } catch (InvalidRequestException e) {
+            errorMessage = e.getMessage();
```

Comment:
We should consider only errorCode not the errorMessage. Message it based in the UI locale which will be resolved in the UI.